code/kopilua/src/lauxlib.cs

```diff
@@ -492,41 +477,39 @@ private static int libsize( luaL_Reg[] l )
 
 
 
-		public static CharPtr luaL_gsub( lua_State L, CharPtr s, CharPtr p,
-																	   CharPtr r )
+		public static string luaL_gsub( lua_State L, string s, string p, string r )
```

Comment:
Check if this is even needed. Might just replace it with C# string replacement function